packages/@uppy/webcam/src/Webcam.jsx

```diff
@@ -99,6 +99,9 @@ export default class Webcam extends UIPlugin {
       mirror: true,
       showVideoSourceDropdown: false,
       facingMode: 'user', // @TODO: remove in the next major
+      videoConstraints: {
+        facingMode: 'user',
+      },
```

Comment:
We should keep it as `undefined`, otherwise it would be a breaking change for users that rely on the deprecated option.
```suggestion
      videoConstraints: undefined,
```

Reply:
But the deprecated option `facingMode` is set to `user`?

https://github.com/transloadit/uppy/pull/4070/files#diff-1bac53cb71c621a71635fc599a8fea3ce9b5411f82024432110848304f28c6e6R101

Reply:
https://github.com/transloadit/uppy/blob/b78937f2bccce141c340a11c35bcb5ebe641f6a5/packages/@uppy/webcam/src/Webcam.jsx#L604

If `videoConstraints?.facingMode` is truthy, it's going to ignore the value of `facingMode`. If the default value of `videoConstraints` is `{ facingMode: 'user' }` and the user has in their code `.use(Webcam, { facingMode: 'environment' }`, we're going to read the default value of `videoConstraints` and completly ignore the deprecated option. The only way to use the deprecated option would be for the user to do `.use(Webcam, { facingMode: 'environment', videoConstraints: { facingMode: null }})` which is not right.